spec/lib/myfinance/request_spec.rb

```diff
@@ -2,10 +2,10 @@
 
 describe Myfinance::Request do
   describe "#run" do
-    subject { described_class.new({ method: 'post', params: {}, url: Myfinance.configuration.url, user_agent: 'My Test User-Agent', authorization_hash: 'my-auth-hash' }) }
+    subject { described_class.new({ method: 'post', params: {}, url: Myfinance.configuration.url, user_agent: 'My Test User-Agent', token: 'my-auth-hash' }) }
```

Comment:
Line is too long. [158/80]<br>Redundant curly braces around a hash parameter.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.